src/wp-includes/blocks/index.php

```diff
@@ -6,67 +6,9 @@
-require ABSPATH . WPINC . '/blocks/comment-reply-link.php';
-require ABSPATH . WPINC . '/blocks/comment-template.php';
-require ABSPATH . WPINC . '/blocks/comments-pagination-next.php';
-require ABSPATH . WPINC . '/blocks/comments-pagination-numbers.php';
-require ABSPATH . WPINC . '/blocks/comments-pagination-previous.php';
-require ABSPATH . WPINC . '/blocks/comments-pagination.php';
-require ABSPATH . WPINC . '/blocks/comments-title.php';
-require ABSPATH . WPINC . '/blocks/cover.php';
-require ABSPATH . WPINC . '/blocks/file.php';
-require ABSPATH . WPINC . '/blocks/gallery.php';
-require ABSPATH . WPINC . '/blocks/home-link.php';
-require ABSPATH . WPINC . '/blocks/image.php';
-require ABSPATH . WPINC . '/blocks/latest-comments.php';
-require ABSPATH . WPINC . '/blocks/latest-posts.php';
 require ABSPATH . WPINC . '/blocks/legacy-widget.php';
-require ABSPATH . WPINC . '/blocks/loginout.php';
-require ABSPATH . WPINC . '/blocks/navigation-link.php';
-require ABSPATH . WPINC . '/blocks/navigation-submenu.php';
-require ABSPATH . WPINC . '/blocks/navigation.php';
-require ABSPATH . WPINC . '/blocks/page-list.php';
-require ABSPATH . WPINC . '/blocks/pattern.php';
-require ABSPATH . WPINC . '/blocks/post-author-biography.php';
-require ABSPATH . WPINC . '/blocks/post-author.php';
-require ABSPATH . WPINC . '/blocks/post-comments.php';
-require ABSPATH . WPINC . '/blocks/post-comments-form.php';
-require ABSPATH . WPINC . '/blocks/post-content.php';
-require ABSPATH . WPINC . '/blocks/post-date.php';
-require ABSPATH . WPINC . '/blocks/post-excerpt.php';
-require ABSPATH . WPINC . '/blocks/post-featured-image.php';
-require ABSPATH . WPINC . '/blocks/post-navigation-link.php';
-require ABSPATH . WPINC . '/blocks/post-template.php';
-require ABSPATH . WPINC . '/blocks/post-terms.php';
-require ABSPATH . WPINC . '/blocks/post-title.php';
-require ABSPATH . WPINC . '/blocks/query-no-results.php';
-require ABSPATH . WPINC . '/blocks/query-pagination-next.php';
-require ABSPATH . WPINC . '/blocks/query-pagination-numbers.php';
-require ABSPATH . WPINC . '/blocks/query-pagination-previous.php';
-require ABSPATH . WPINC . '/blocks/query-pagination.php';
-require ABSPATH . WPINC . '/blocks/query-title.php';
-require ABSPATH . WPINC . '/blocks/query.php';
-require ABSPATH . WPINC . '/blocks/read-more.php';
-require ABSPATH . WPINC . '/blocks/rss.php';
-require ABSPATH . WPINC . '/blocks/search.php';
-require ABSPATH . WPINC . '/blocks/shortcode.php';
-require ABSPATH . WPINC . '/blocks/site-logo.php';
-require ABSPATH . WPINC . '/blocks/site-tagline.php';
-require ABSPATH . WPINC . '/blocks/site-title.php';
-require ABSPATH . WPINC . '/blocks/social-link.php';
-require ABSPATH . WPINC . '/blocks/tag-cloud.php';
-require ABSPATH . WPINC . '/blocks/template-part.php';
-require ABSPATH . WPINC . '/blocks/term-description.php';
 require ABSPATH . WPINC . '/blocks/widget-group.php';
+require ABSPATH . WPINC . '/blocks/require-blocks.php';
```

Comment:
Nit: Let's move `/blocks/require-blocks.php` as the first import before all special cases.